parsec/backend/organization.py

```diff
@@ -242,6 +244,25 @@ async def api_organization_bootstrap(
                 "reason": "Redacted user&device certificate muste be provided together",
             }
 
+        # Sequester can not be set with APIV1
+        if isinstance(client_ctx, APIV1_AnonymousClientContext):
+            sequester_verify_key_certificate = None
+        else:
+            sequester_verify_key_certificate = msg["sequester_verify_key_certificate"]
+            if sequester_verify_key_certificate is not None:
+                try:
+                    sequester_verify_key_certificate = SequesterVerifyKeyCertificate.verify_and_load(
+                        sequester_verify_key_certificate,
+                        author_verify_key=root_verify_key,
+                        expected_author=None,
+                    )
+
+                except DataError:
+                    return {
+                        "status": "invalid sequester_verify_key_certificate",
```

Comment:
no space in status code, should be `invalid_sequester_verify_key_certificate`

And why not using the regular `invalid_certification`/`invalid_data` errors instead of a custom one ?

Reply:
renamed to `invalid_data`